packages/modules/src/BatchedSessionRouterModule.ts

```diff
@@ -100,7 +100,9 @@ export class BatchedSessionRouterModule extends BaseValidationModule {
   async signUserOpHash(userOpHash: string, params?: ModuleInfo): Promise<string> {
     const sessionParams = params?.batchSessionParams
     if (!sessionParams || sessionParams.length === 0) {
-      throw new Error('Session parameters are not provided.')
+      throw new Error(
+        'Session parameters are not provided or of invalid length. Only one object is allowed in sessionParams array.'
```

Comment:
In case of BatchedSessionRouter module, this error message doesn't apply here.

Only one object is allowed in sessionParams array

Now we have batchSessionParams which is expected to be just non-empty.

Reply:
ok i see! done